src/explore-education-statistics-common/src/modules/table-tool/components/TableHeadersGroupControls.tsx

```diff
@@ -49,6 +52,17 @@ const TableHeadersGroupControls = ({
   const displayItems = isExpanded ? totalItems : defaultNumberOfItems;
   const hasMoreItems = isExpanded || totalItems > displayItems;
   const disableControls = activeGroup && activeGroup !== id;
+  const { getValues, setValue } = useFormContext();
+  const list: Filter[] = getValues(groupName);
+  const [isReverse, setIsReverse] = useState(false);
+
+  useEffect(() => {
```

Comment:
I don't think this needs to be in a `useEffect`, you could just trigger it on click directly, like:
```
  const handleReverse = () => {
    setValue(groupName, list.toReversed());
    toggleReverseOrder(id);
  };
  ```